tests/arxml/expected_gen/portinterface/ar4_client_server_interface_single_operation_no_return_with_desc.arxml

```diff
@@ -0,0 +1,402 @@
+<?xml version="1.0" encoding="utf-8"?>
+<AUTOSAR xmlns="http://autosar.org/schema/r4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://autosar.org/schema/r4.0 AUTOSAR_4-2-2.xsd">
+  <AR-PACKAGES>
+    <AR-PACKAGE>
+      <SHORT-NAME>DataTypes</SHORT-NAME>
```

Comment:
This file is very large, you can reduce file size in your unit test by saving just the XML under test.
In this particular case you can use the filters=['/PortInterfaces'] as extra argument in your call to self.save_and_check in ar4_portinterface_test.py to save just the PortInterfaces package.

Reply:
Shorter file is better. Fixed.